guides/common/assembly_configuring-capsule-custom-server-certificate.adoc

```diff
@@ -11,8 +11,12 @@ To configure your {SmartProxyServer} with a custom certificate, complete the fol
 . xref:deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[]
 . xref:deploying-a-custom-ssl-certificate-to-hosts_{smart-proxy-context}[]
 
+:parent-ProductName: {SmartProxyServer}
```

Comment:
```suggestion
:parent-ProductName: {ProductName}
:ProductName: {SmartProxyServer}
```

With this, you can use the "{ProductName}" attribute in your module as expected

Reply:
Oh yes! That's a great idea!